vcs/utils.py

```diff
@@ -2256,6 +2256,8 @@ def creategraphicsmethod(gtype, gname='default', name=None):
         func = vcs.create1d
     elif gtype in ['vector', 'Gv']:
         func = vcs.createvector
+    elif gtype in ['streamline', 'Gs']:
+        func = vcs.createstreamline
```

Comment:
@danlipsa had forgotten it in his streamline branch. When i ran my test on streamline it couldn't generate the gm so I fixed it here